uint16 entity_id
string entity_name
float64 latitude
float64 longitude
float64 altitude
float64 heading
float64 pitch
float64 roll
float64 velocity_x
float64 velocity_y
float64 velocity_z
uint8 force_id
uint32 timestamp